### ISSUE TITLE
Symbol Lookup Error

### ISSUE THREAD

**Issue by varad-kadam** [badabing2005/PixelFlasher#109]
Hello, 
I upgraded to PixelFlasher Version 5.8.2.0 and am encountering Android Platform Tools issues.

System:
EndeavourOS Linux: Kernel `6.5.9-arch1-1-g14`
Platform Tools: Tested `platform-tools_r34.0.5-linux` and `platform-tools_r33.0.3-linux`
PixelFlasher: Tested on `5.8.2.0` and `5.8.1.0`

Other Info:
Device: `Pixel 7`
Image: `panther-up1a.231105.003-factory-1f4e21fc.zip`

Console (for both versions):
```
2023-11-10 13:44:20 Selected Platform Tools Path:
/home/user/Apps/platform-tools.

2023-11-10 13:44:20 ERROR: /bin/sh: symbol lookup error: /bin/sh: undefined symbol: rl_trim_arg_from_keyseq


2023-11-10 13:44:20 ERROR: Android Platform Tools version is not available or is too old.
                           For your protection, disabling device selection.
                           Please select valid Android SDK.

SDK Version:      None
Adb SHA256:       d58bfae1297bed30780608dde3a1ffe1be4efc7c99512edef136c7c25c34eb1e
Fastboot SHA256:  08d0f9f73405854401208e01657d10683818b0ccf8ab548d4a287135bb0b3e15
```

Any ideas as to what might be going wrong, or any more information that might be needed?

**Comment by badabing2005** [comment]
See if this helps
https://xdaforums.com/t/pixelflasher-a-gui-tool-for-flashing-updating-rooting-managing-pixel-phones.4415453/post-89148382

**Comment by varad-kadam** [comment]
Yup, that fixed the problem, thanks!

**Comment by fabifont** [comment]
> Yup, that fixed the problem, thanks!

I'd like to request that this issue be reopened, as it stems from an outdated version of `libreadline` used by `pixelflasher`. The current version is incompatible with all versions of bash starting from `5.2` (https://bugs.freebsd.org/bugzilla//show_bug.cgi?id=266762), leading to this problem.

A temporary solution involves downgrading bash from `5.2` to `5.1`, but this is not an ideal fix.

@badabing2005

**Comment by badabing2005** [comment]
I'll reopen it, but realize that you're using an Ubuntu build on non Ubuntu OS
Pixelflasher is build using Github actions using the [available / supported OSes](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) by the runners.

Ubuntu22.04 which is the latest supported on Github, only bundles libreadline 8.1.2-1 which is older than what you need and hence why it breaks with your newer OS.

Can you please try this and tell me if it works.
- Launch PixelFlasher, don't set the platform tools yet, ignore the error it generates 
- assuming that the app is still running, open a terminal and navigate to /tmp
- You should find a directory that starts with `_MEI*`
- cd to that directory and delete libreadline* `rm libreadline*`
- Then try to select Android Platform tools in PixelFlasher.
- If it works, I'll add an option to delete the file on every startup.

**Comment by varad-kadam** [comment]
>     * Launch PixelFlasher, don't set the platform tools yet, ignore the error it generates
> 
>     * assuming that the app is still running, open a terminal and navigate to /tmp
> 
>     * You should find a directory that starts with `_MEI*`
> 
>     * cd to that directory and delete libreadline* `rm libreadline*`
> 
>     * Then try to select Android Platform tools in PixelFlasher.

Yes, I can confirm this works on my end.

**Comment by badabing2005** [comment]
@varad-kadam Thank you for confirming.
I'll add this to the next build.

**Comment by badabing2005** [comment]
Issue should be addressed in PixelFlasher 6.0.0.0
Before you select the Platform tools, go to settings and add any library that you want PixelFlasher to not load.
In this case it would be `libreadline.so.8` you can also use wildcards like `libreadline*` and specify multiple libraries (comma separated).
